nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -563,6 +563,15 @@
        earlier version of NixOS.
      </para>
    </listitem>
+   <listitem>
+     <para>
+       <literal>linux_5_1</literal>, <literal>linux_5_2</literal> and <literal>linux_5_3</literal> package
+       attributes have been removed to discourage dependence on specific non-LTS kernel versions in stable
+       NixOS releases. Please use an LTS kernel package, <literal>linux_latest</literal> or
+       <literal>linux_testing</literal>. For more details see
+       <link xlink:href="https://github.com/NixOS/nixpkgs/pull/70202">PR #70202</link>
```

Comment:
Umm, I really don't feel so great about linking PR's in the release notes for context. Can whatever context that could be added be explained here?

Reply:
I somewhat agree, even if this has the most unfiltered information about the whys (like in https://logs.nix.samueldr.com/nixos-dev/2019-10-07#1570476520-1570473549;).

It feels to me like there should be 1-2 paragraphs in the kernel section in the manual about supported kernel versions and pitfalls. But meanwhile, I'll try to see if I can come up with a short half-sentence to replace the PR link without going to deep.

Reply:
@NeQuissimus @worldofpeace: I tried to elaborate a little bit on the intention and improve phrasing. Let me know what you think.